Automattic/wp-calypso
# State: Introduce isSiteVip selector

While working on getting `sites-list` out of `MapDomain` in #12804, @nb introduced a `isSiteVip` selector. The refactor landed in another PR (#12943), so we closed #12804, but the `isSiteVip` selector could be the safer choice in various cases where we use `site.is_vip`. We already have selectors for similar cases (`isJetpackSite` for `site.jetpack`), so having one for VIP makes sense.

I've addressed a couple of small suggestions that were recommended in #12804, so this one is ready to review.

To test:
* Checkout this branch.
* Verify all selector tests pass:

```
npm run test-client client/state/selectors/
```
